lib/index.ts

```diff
@@ -169,15 +178,27 @@ export function persistTree(structure : any, store : any, debug: boolean) {
      * ever need to unsubscribe from state updates.
      */
     store.subscribe(handleChange);
+    _loadAutomaticReducers(store);
 
     // If debug, we to log all the actions for loading the state
-    if(debug) {
-        _map(structure, async (object : any, name : any) => {
+    if (debug) {
+        _map(structure, async (object: any, name: any) => {
             console.log(object.action || buildAction(name));
         });
     }
 }
 
+function _loadAutomaticReducers(store: any) {
```

Comment:
Looking good! 

Could we remove the `_` from this function, and just have it be called `loadAutomaticReducers`?

Also, would you be able to add comments to it, etc:



```
/**
* What does this function do?
*/
function loadAutomaticReducers(store: any) {

```